pages/open_cas_linux/guide_terminology.md

```diff
@@ -0,0 +1,33 @@
+| cache hit          | When requested data is contained in (and returned from) the cache.                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| cache miss         | When requested data is not in the cache, and therefore must be retrieved from its primary storage location.                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| core device        | The device to be cached.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| dirty data         | This refers to data that is modified within the cache but not modified in main memory.                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| guest              | An operating system running on a Virtual Machine (VM) environment.                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
Words "guest", "host" and "hypervisior" do not occur on any page of this documentation. (At least not in meaning described here).